### PR TITLE
CASH-919 Add half-star ratings for lend filter

### PR DESCRIPTION
**Pull request by foochris**

### CODE REVIEW

File: src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue [hunk 1]
```diff
@@ -113,7 +122,7 @@ export default {
 			return Math.max(this.minimum, this.range.min);
 		},
 		sliderMaximum() {
-			return Math.min(this.maximum, this.range.max);
+			return this.attribute === 'partner.riskRating' ? this.maximum : Math.min(this.maximum, this.range.max);
```
Comment (ksekhri) [inline src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue]: This change actually breaks the UI in situations where not all star ratings are available. Try limiting the set to 10/15 loans then drag the slider, the handle and the progress bar don't match. Can we revert?

File: src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue [hunk 2]
```diff
@@ -72,6 +73,14 @@ export default {
 			type: String,
 			required: true
 		},
+		step: {
+			type: Number,
+			default: 1
+		},
+		attribute: {
```
Comment (ksekhri) [inline src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue]: If we revert line 125 can we remove this?

File: src/pages/Lend/Filter/FilterSections/LoanDetails/RangeSliderWrapper.vue [hunk 3]
```diff
@@ -16,7 +17,9 @@
 				:is-slider-setting-minimum="isSliderSettingMinimum"
 				:range="range"
 				:refine="refine"
+				:step="step"
 				:title="title"
+				:attribute="queryAttribute"
```
Comment (ksekhri) [inline src/pages/Lend/Filter/FilterSections/LoanDetails/RangeSliderWrapper.vue]: Same question about removing

File: src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue (hunk 1 above)
Comment (foochris) [reply src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue]: @ksekhri Sure thing! Reverted.

**Pull request merged**